app/views/users/two_factor_authentication/setup/show_email.html.erb

```diff
@@ -0,0 +1,18 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-full">
+    <h2 class="govuk-heading-l">Two-factor authentication</h2>
+    <p class="govuk-body">
+      We can send security codes to the email address associated with your account. Complete the
+      setup below to start receiving security codes by email from now on.
+    </p>
+    <p class="govuk-body">
+      Alternative, go back if you wish to receive security codes in another way.
+    </p>
+    <p class="govuk-body">
+      <%= form_tag "/users/two_factor_authentication/setup?method=email", method: :put do %>
```

Comment:
Ideally this would be a hidden `input` with `name="method" value="email"` rather than exposing the interface in the URL.

Reply:
I was being lazy, thanks for pointing this out!